python/cudf/cudf/datasets.py

```diff
@@ -2,14 +2,20 @@
 import pandas as pd
 
 import cudf
+from cudf._lib.transform import bools_to_mask
 
 __all__ = ["timeseries", "randomdata"]
 
 
 # TODO:
 # change default of name from category to str type when nvstring are merged
 def timeseries(
-    start="2000-01-01", end="2000-01-31", freq="1s", dtypes=None, seed=None,
+    start="2000-01-01",
+    end="2000-01-31",
+    freq="1s",
+    dtypes=None,
+    nulls_frequency=0.1,
```

Comment:
Should the default be 0 or should we always be generating null data ?

Reply:
Ah yes, to maintain the promise of `non-breaking` it should default to 0.